definitions/procedures/pulp/remove.rb

```diff
@@ -64,18 +65,25 @@ def ask_to_proceed(rm_cmds)
 
     def run
       rm_cmds = data_dir_removal_cmds
-      ask_to_proceed(rm_cmds) if rm_cmds.any?
+
+      assumeyes_val = @assumeyes.nil? ? assumeyes? : @assumeyes
+
+      ask_to_proceed(rm_cmds) unless assumeyes_val
 
       remove_pulp
```

Comment:
Better to not run `remove_pulp` itself if `pulp_packages` is empty? because the question wont include `rpm -e` command we will still display removal message?

Reply:
@jjeffers I wished to have `if pulp_packages.any?` here and remove it from `remove_pulp`,

~~~
   def remove_pulp
      with_spinner('Removing pulp2 packages') do
        execute!("rpm -e #{pulp_packages.join('  ')}")
      end
    end
~~~